<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="FrontAccounting" duration-ms="93035" started-at="2018-12-30T15:36:00Z" finished-at="2018-12-30T15:37:33Z">
    <groups>
    </groups>
    <test name="Front Accounting TestCases" duration-ms="93035" started-at="2018-12-30T15:36:00Z" finished-at="2018-12-30T15:37:33Z">
      <class name="com.test.testcases.SalesPersonsPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.testcases.SalesPersonsPageTest@1e127982]" name="setUp" is-config="true" duration-ms="10631" started-at="2018-12-30T15:37:11Z" finished-at="2018-12-30T15:37:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifySalesPersonsTittle()[pri:1, instance:com.test.testcases.SalesPersonsPageTest@1e127982]" name="verifySalesPersonsTittle" duration-ms="19" started-at="2018-12-30T15:37:22Z" finished-at="2018-12-30T15:37:22Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Sals Persons] but found [Sales Persons]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Sals Persons] but found [Sales Persons]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.test.testcases.SalesPersonsPageTest.verifySalesPersonsTittle(SalesPersonsPageTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySalesPersonsTittle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.testcases.SalesPersonsPageTest@1e127982]" name="tearDown" is-config="true" duration-ms="10660" started-at="2018-12-30T15:37:23Z" finished-at="2018-12-30T15:37:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.testcases.SalesPersonsPageTest -->
      <class name="com.test.testcases.LoginPageTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.test.testcases.LoginPageTest@6e1ec318]" name="setup" is-config="true" duration-ms="13083" started-at="2018-12-30T15:36:00Z" finished-at="2018-12-30T15:36:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="loginTitleTest()[pri:1, instance:com.test.testcases.LoginPageTest@6e1ec318]" name="loginTitleTest" duration-ms="40" started-at="2018-12-30T15:36:13Z" finished-at="2018-12-30T15:36:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.testcases.LoginPageTest@6e1ec318]" name="tearDown" is-config="true" duration-ms="15672" started-at="2018-12-30T15:36:14Z" finished-at="2018-12-30T15:36:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.testcases.LoginPageTest -->
      <class name="com.test.testcases.SalesTypesPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.testcases.SalesTypesPageTest@617faa95]" name="setUp" is-config="true" duration-ms="7953" started-at="2018-12-30T15:36:52Z" finished-at="2018-12-30T15:37:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifySalesTypesTittle()[pri:1, instance:com.test.testcases.SalesTypesPageTest@617faa95]" name="verifySalesTypesTittle" duration-ms="14" started-at="2018-12-30T15:37:00Z" finished-at="2018-12-30T15:37:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Sales ypes] but found [Sales Types]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Sales ypes] but found [Sales Types]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.test.testcases.SalesTypesPageTest.verifySalesTypesTittle(SalesTypesPageTest.java:54)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySalesTypesTittle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.testcases.SalesTypesPageTest@617faa95]" name="tearDown" is-config="true" duration-ms="10662" started-at="2018-12-30T15:37:01Z" finished-at="2018-12-30T15:37:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.testcases.SalesTypesPageTest -->
      <class name="com.test.testcases.HomePageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.testcases.HomePageTest@7e0b0338]" name="setUp" is-config="true" duration-ms="7133" started-at="2018-12-30T15:36:29Z" finished-at="2018-12-30T15:36:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyHomePageTitleTest()[pri:1, instance:com.test.testcases.HomePageTest@7e0b0338]" name="verifyHomePageTitleTest" duration-ms="30" started-at="2018-12-30T15:36:36Z" finished-at="2018-12-30T15:36:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomePageTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.testcases.HomePageTest@7e0b0338]" name="tearDown" is-config="true" duration-ms="15634" started-at="2018-12-30T15:36:36Z" finished-at="2018-12-30T15:36:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.testcases.HomePageTest -->
    </test> <!-- Front Accounting TestCases -->
  </suite> <!-- FrontAccounting -->
</testng-results>
